### PR TITLE
About Pet Adoption Page

### PR DESCRIPTION
**Pull request by JHump20**
Created a pet Adoption about page

### CODE REVIEW

**Comment by JHump20** [comment]
Not sure if this worked please confirm someone.

**Comment by SpencerBellucci** [comment]
Hey JHump, the only thing I see that should be changed is the addition of the navbar to go to other pages. all add comments to code with where additional code should be

File: AboutPage.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html>
+<head>
+	<title>About Pet Adoption></title> 
+	<meta charset="utf8">
```
Comment (SpencerBellucci) [inline AboutPage.html]: add this here "<link rel="stylesheet" href="style.css">"

File: AboutPage.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+</head>
+<body>
```
Comment (SpencerBellucci) [inline AboutPage.html]: and add this here "
        <div class="nav">
            <a class="active" href="main.html">Home</a>
            <a href="pets.html">Pets</a>
            <a href="adoption.html">Adoption</a>
            <a href="about.html">About</a>
        </div>"

Comment (SpencerBellucci) [reply AboutPage.html]: that turned into real links, lemme fix that,

        <div class="nav">
            <a class="active" href="main.html">Home</a>
            <a href="pets.html">Pets</a>
            <a href="adoption.html">Adoption</a>
            <a href="about.html">About</a>
        </div>-->

File: AboutPage.html (hunk 1 above)
Comment (SpencerBellucci) [reply AboutPage.html]: sorry gets written in

File: AboutPage.html (hunk 2 above)
Comment (SpencerBellucci) [inline AboutPage.html:7]: add this here

        <div class="nav">
            <a class="active" href="main.html">Home</a>
            <a href="pets.html">Pets</a>
            <a href="adoption.html">Adoption</a>
            <a href="about.html">About</a>
        </div>

File: AboutPage.html (hunk 1 above)
Comment (SpencerBellucci) [inline AboutPage.html]: add this here

 <link rel="stylesheet" href="style.css">

Comment (SpencerBellucci) [inline AboutPage.html:5]: add this
```<link rel="stylesheet" href="style.css">```

**Comment by JHump20** [comment]
I believe I added it

**Pull request merged**